### ISSUE TITLE
[Feature] – Return transitive dependencies list in alphabetical order

### ISSUE THREAD

**Issue by douglasgondim** [Nikoloutsos/explicitDependencyImportCheck#2]
It would be nice if we had this feature to have consistency in the results.

**Comment by kostasniks** [comment]
Yes, we can sort the transitive dependencies alphabetically for same error reporting each time 👍 
I am more than welcome if you want to contribute to this repo by creating a PR

**Comment by Nikoloutsos** [comment]
Yes, we can sort the transitive dependencies alphabetically for same error reporting each time 👍
I am more than welcome @douglasgondim  if you want to contribute to this repo by creating a PR
